Kiosk/App/AppDelegate.swift

```diff
@@ -5,19 +5,26 @@ import RxSwift
 import Keys
 import Stripe
 
-@UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate {
-    
+@UIApplicationMain class AppDelegate: UIResponder, UIApplicationDelegate {
+
     let helpViewController = Variable<HelpViewController?>(nil)
     var helpButton: UIButton!
 
     weak var webViewController: UIViewController?
 
-    var window: UIWindow? = UIWindow(frame:CGRect(x: 0, y: 0, width: UIScreen.main.bounds.height, height: UIScreen.main.bounds.width))
+    var window: UIWindow? = UIWindow(frame: CGRect(
+        x: 0,
+        y: 0,
+        width: UIScreen.main.bounds.height,
+        height: UIScreen.main.bounds.width
+    ))
 
-    fileprivate(set) var provider = Networking.newDefaultNetworking()
+    fileprivate (set) var provider = Networking.newDefaultNetworking()
```

Comment:
Honestly not sure, I couldn't find any examples in swift style guides. I feel like `fileprivate(set)` it what I'd expect, @ashfurrow?

Reply:
I like the "space after all keywords" rule that Prettier ([almost](https://github.com/prettier/prettier/issues/1139) follows.

Reply:
Did some research, this one is interesting, because all [the apple docs](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/AccessControl.html) have no space.

However, it compiles fine both ways and I like the language consistency that's used in the issue you mentioned.

Reply:
The reason no space works better for me is that `fileprivate` *only* affects the `set`, not the whole declaration. That's clearer without the whitespace. (Also, this is a cool effort!)

Reply:
Thought of another way, `(set)` modifies `fileprivate`.